app/services/idv/in_person_config.rb

```diff
@@ -6,7 +6,7 @@ def self.enabled_for_issuer?(issuer)
       if issuer.nil?
         enabled_without_issuer?
       else
-        enabled_issuers.include?(issuer)
+        ServiceProvider.find_by(issuer: issuer)&.in_person_proofing_enabled || false
```

Comment:
I was thinking about it before, but since you're bringing it up I decided to go ahead with this instead:

```ruby
ServiceProvider.exists?(
  issuer: issuer,
  in_person_proofing_enabled: true
)
```